### PR TITLE
Add group for pypi package

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cciutils/publish.py [hunk 1]
```diff
@@ -7,43 +7,38 @@
...
     Publish to pypi
 
     version_type: Describe the kind of release we do: rebuild (specified using --type), version_tag,
                   version_branch, feature_branch, feature_tag (for pull request)
     publish: If False only check the package
-    config is like:
-        packages:
-          - path: . # the root foder of the package
+    package is like:
+        path: . # the root foder of the package
```
Comment (arnaud-morvan) [inline c2cciutils/publish.py:18]:
```suggestion
        path: . # the root folder of the package
```

File: c2cciutils/scripts/publish.py [hunk 2]
```diff
@@ -114,14 +114,16 @@ def main() -> None:
 
     success = True
     pypi_config = config.get("publish", {}).get("pypi", {})
-    success &= c2cciutils.publish.pip(
-        pypi_config, version, version_type, version_type in pypi_config.get("versions", [])
-    )
+    for package in pypi_config["packages"]:
+        if package.get("group", "") == args.group:
```
Comment (arnaud-morvan) [inline c2cciutils/scripts/publish.py:118]: Note: 
It seems that we have "default" as a default in package group config.
You also have "default" as a default for group parameter.
Note sure we need a default "" here.

**Review by arnaud-morvan — APPROVED** [review]
Seems you cancelled the twine check.

File: c2cciutils/scripts/publish.py (hunk 2 above)
Comment (sbrunner) [reply c2cciutils/scripts/publish.py:118]: Effectively no :-)

**Pull request merged**